pkg/patterns/declarative/pkg/applier/type.go

```diff
@@ -5,14 +5,15 @@ import (
 
 	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/client-go/rest"
+	"sigs.k8s.io/kubebuilder-declarative-pattern/pkg/patterns/declarative/pkg/manifest"
 )
 
 type Applier interface {
 	Apply(ctx context.Context, options ApplierOptions) error
 }
 
 type ApplierOptions struct {
-	Manifest string
+	Objects []*manifest.Object
```

Comment:
Do you mind explaining a little bit why this replaces the Manifests with `manifest.Object` slice?  I look at all the places where use `ApplierOption.Objects`, it seems to have no logic change besides the two types conversion. 

From a style perspective, I like the change (typed resource group vs serialized value), just to ask to double check in case I miss any feature changes.

Reply:
That was actually https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/pull/275 , it was included here because github doesn't have chained PRs.  Sorry - I should have said "builds on #275" or similar.

The actual problem was that I was passing yaml to something that expected json.  They are both `string` (or `[]byte`).  To prevent that confusion in future, I switched to typed objects.

Reply:
That's fair! Thanks for the explanation.